### PR TITLE
MULTISITE-12704 Make file name editable

### PR DESCRIPTION
**Pull request by ulrika-lolliot**

### CODE REVIEW

File: lib/features/farnet_core/farnet_core.info [hunk 1]
```diff
@@ -66,6 +66,7 @@ features[field_base][] = field_website
 features[field_base][] = field_zip_code
 features[field_base][] = field_latitude
 features[field_base][] = field_longitude
+features[field_base][] = filename_field
```
Comment (bianchim) [inline lib/features/farnet_core/farnet_core.info]: The field instance is missing from the info file

**Comment by bianchim** [comment]
The file field widget should be changed to a media browser, currently when uploading a document you can't directly change its name.

**Comment by ulrika-lolliot** [comment]
The widget should already be media browser, there must be something wrong in the feature recreation then. I'll check that.

**Comment by ulrika-lolliot** [comment]
I rebuilt my instance completely and I have media browser as widget type for a file field of Farnet Event (in the field collection Speaker), and i can edit the file name. Can you try disabling and enabling the feature?

**Comment by bianchim** [comment]
Rebuilding the site from scratch fixed the problem, merging

**Pull request merged**